Documentation/SwiftSyntaxBuilder.md

```diff
@@ -0,0 +1,5 @@
+# Swift Syntax Builder
```

Comment:
I’m not sure if this is really providing any value or if we should just remove it.